### DISCUSSION TITLE
QQ群好冷清啊

### DISCUSSION THREAD

**Discussion by YuTinMIn** [share121/inter-knot#3055] [其他]
![微信截图_20240819152124](https://github.com/user-attachments/assets/d7ea4cce-d751-4bd5-83ae-d744a57cdcab)
有没有好心人加一下这个群里的人都似了的群，就8个人（准确来说是5个人，有3个号是我的）

**Comment by SeeieKianna** [comment]
好了，这个委托我接了

**Reply by peng-shao-stay** [reply]
我来实现愿望

**Comment by vanicmobeus** [comment]
真是拿你没办法，那我进来了哦（蜀蜀脸

**Comment by Jiucheng258** [comment]
群里真的有猫希人女仆吗？

**Comment by RaphaelHider** [comment]
验证问题答案呢（恼）

**Reply by RaphaelHider** [reply]
> 验证问题答案呢（恼）

不好意思是我的问题，扫码就不用回答问题，我是用pc搜索群号才需要验证

**Comment by warning-Kirov-reporting** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/07/31/fdc5e0e30e577fb208f29effa93510ed_8442612635207222552.png" alt="云璃 期待" width="75" >我来！！！

**Comment by shan-dgha** [comment]
没办法的那<img src="https://bbs-static.miyoushe.com/static/2024/01/31/f8a6716e3f0ab4d3235fbb8fd50e481f_2873815640813891077.png" alt="花火 点火" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/01/31/f8a6716e3f0ab4d3235fbb8fd50e481f_2873815640813891077.png" alt="花火 点火" width="75" >

**Comment by YuTinMIn** [comment]
如果是QQ号加群的，答案：云天明YuTinMIn  （其实就是我的网名），注意大小写！！
